client/src/modules/dashboard/Venues/pages/VenuePage.tsx

```diff
@@ -26,6 +48,18 @@ export const VenuePage: NextPageWithLayout = () => {
   if (!data.venue)
     return <NextError statusCode={404} title="Venue not found" />;
 
+  const chapterId = data.venue.chapter.id;
+
+  const clickDelete = async () => {
+    const ok = await confirmDelete({
+      body: 'Are you sure you want to delete this venue? All information related to venue will be deleted. Venue deletion cannot be reversed.',
+      buttonText: 'Delete Venue',
+    });
+    if (!ok) return;
+    deleteVenue({ variables: { venueId, chapterId } });
+    router.push('/dashboard/venues');
```

Comment:
```suggestion
    router.replace('/dashboard/venues');
```
Replaces the url entry, making it impossible to use _back button_ to go to the venue page for deleted venue.

Reply:
I remember, you or oliver telling me this before, I should have known better 😞